frontend/styles/maps.scss

```diff
@@ -61,5 +61,11 @@ $utilities: (
     responsive: true,
     property: position,
     values: static relative absolute fixed sticky
+  ),
+  'max-width': (
```

Comment:
can't find these classes used anywhere. should we remove them then?

Reply:
they're here: https://github.com/okfde/froide/blob/bikubi/postupload-fixes/frontend/javascript/components/redaction/pdf-redaction.vue#L164 `mw-lg-50 mw-xl-25`